django_project/ford3/static/js/provider_form.js

```diff
@@ -1,14 +1,18 @@
-var slider = document.getElementById('campus-count');
 var output = document.getElementById('number-of-campuses');
-slider.value = 1;
-output.value = slider.value; // Display the default slider value
 // Update the current slider value (each time you drag the slider handle)
-slider.oninput = function () {
-    output.value = this.value;
-
-    createCampusNameInputs(this.value);
+output.onchange = function () {
+    createCampusNameInputs(output.value);
```

Comment:
What about your idea of having only one campus input text and a button `add campus` that will add a new input text.